plugins/inputs/knx_listener/README.md

```diff
@@ -0,0 +1,66 @@
+# KNX input plugin
+
+The KNX input plugin that listens for messages on the KNX home-automation bus.
+This plugin connects to the KNX bus via a KNX-IP interface.
+Information about supported KNX message datapoint types can be found at the
+underlying "knx-go" project site (https://github.com/vapourismo/knx-go).
+
+### Configuration
+
+This is a sample config for the plugin.
+
+```toml
+# Listener capable of handling KNX bus messages provided trough a KNX-IP Interface.
+[[inputs.KNXListener]]
+  # Type of KNX-IP interface.
+  # Can be either "tunnel" or "router".
+  service_type = "tunnel"
+
+  # Address of the KNX-IP interface.
+  service_address = "localhost:3671"
```

Comment:
`have two # for comments` -> https://github.com/influxdata/telegraf/wiki/Review#configuration
(also, sync with `SampleConfig()`)

Reply:
Fixed, can be resolved